cli/src/main/java/com/devonfw/tools/ide/variable/IdeVariables.java

```diff
@@ -55,9 +54,12 @@ public interface IdeVariables {
   VariableDefinitionString JASYPT_OPTS = new VariableDefinitionString("JASYPT_OPTS", null,
       c -> "algorithm=PBEWITHHMACSHA512ANDAES_256 ivGeneratorClassName=org.jasypt.iv.RandomIvGenerator");
 
+  /** {@link VariableDefinition} for {@link com.devonfw.tools.ide.context.IdeContext#getIdeHome() PROJECT_NAME}. */
+  VariableDefinitionPath PROJECT_NAME = new VariableDefinitionPath("PROJECT_NAME", null, c -> c.getIdeHome().getFileName());

```

Comment:
1. IdeHome may be null causing an NPE here.
2. If the variable is already called `PROJECT_NAME`, I would rather suggest to use `VariableDefinitionString` instead.